kratos/geometries/line_3d_3.h

```diff
@@ -346,15 +340,17 @@ class Line3D3 : public Geometry<TPointType>
     */
     double Length() const override
     {
-        const TPointType& point0 = BaseType::GetPoint(0);
-        const TPointType& point1 = BaseType::GetPoint(2);
-        const double lx = point0.X() - point1.X();
-        const double ly = point0.Y() - point1.Y();
-        const double lz = point0.Z() - point1.Z();
+        Vector temp;
+        const IntegrationMethod integration_method = IntegrationUtilities::GetIntegrationMethodForExactMassMatrixEvaluation(*this);
```

Comment:
This is consistent with 2D quadratic line, see https://github.com/KratosMultiphysics/Kratos/blob/a914af509774260e549a7b1d453757e991258e27/kratos/geometries/line_2d_3.h#L353

This was done by me 4 years ago. I have discussed with @KratosMultiphysics/altair and a priori second order (not 3rd order) should give an exact solution. This may imply that the integration points are wrong. We need to research further but for the moment it will be enough  this PR so at least the Length and other Jacobian related methods are functional